src/main/java/core/basesyntax/generator/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.generator;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder report = new StringBuilder();
```

Comment:
Consider initializing the `StringBuilder` with an initial capacity to optimize performance, especially if the storage is expected to contain a large number of entries.